vmdb/app/models/dialog_field_check_box.rb

```diff
@@ -38,7 +42,7 @@ def default_resource_action
   end
 
   def required_value_error?
-    value != "t"
+    @value != "t"
```

Comment:
This method is depending on the side effect of `value` being called before it.  In other words, if someone calls `required_value_error?` before calling `value`, then `@value` could have a different value.  Why did you change this?